eng/common/scripts/Create-APIReview.ps1

```diff
@@ -172,12 +169,13 @@ function IsApiviewStatusCheckRequired($packageInfo)
     return $false
 }
 
-function ProcessPackage($packageName)
+function ProcessPackage($packageInfo)
 {
     $packages = @{}
     if ($FindArtifactForApiReviewFn -and (Test-Path "Function:$FindArtifactForApiReviewFn"))
     {
-        $packages = &$FindArtifactForApiReviewFn $ArtifactPath $packageName
+        $pkgArtifactName = $packageInfo.ArtifactName ?? $packageInfo.Name
+        $packages = &$FindArtifactForApiReviewFn $ArtifactPath $pkgArtifactName $packageInfo
```

Comment:
We probably should pass these parameters by name so they correctly link-up if one of the functions don't have on or the other of these.